src/layers/conv.jl

```diff
@@ -1452,3 +1452,200 @@ function Base.show(io::IO, l::EGNNConv)
     end
     print(io, ")")
 end
+
+
+@doc raw"""
+    TransformerConv((in, ein) => out; heads=1, concat=true, init=glorot_uniform,
+        add_self_loops=false, bias_qkv=false, bias_root=true, root_weight=true, beta=false)
```

Comment:
`skip_connection`, `batch_norm` and `ff_channels` are missing. 
Since there are so many arguments better to write them here as
```
TransformerConv((in, ein) => out; [heads, concat, init, add_self_loops, bias_qkv, bias_root, root_weight, beta]))
```
and add the defaults to the Arguments section

Reply:
Thanks for pointing this out, I will take care of this!